x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -259,7 +263,7 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
         } catch (e) {}
       }
     },
-    [errors, mutateAsync, packId, watchedValues]
+    [errors, mutateAsync, packId, watchedValues, hideSubmitButton]
   );
   const commands = useMemo(
```

Comment:
if `hideSubmitButton` i think we shouldn't pass `commands`?

Reply:
hideSubmitButton got removed, thanks!